SLR-5 added Prime Numbers in a Range Solution (#3)
ali-imran-pucit/Practice: README.md

### isPrime
* **Input:** Takes an integer as input.
* **Output:** String output, describing if the number is Prime or not.
### primesInRange

> * **Input:** Takes a range of numbers in integar.

Reviewer: ```suggestion
* **Input:** Takes two integer numbers to limit the range.
```
